src/packages.txt

```diff
@@ -11,7 +11,6 @@ tree
 vim
 curl
 openssh-server
-zsh
```

Comment:
😢

Reply:
If you need it I will drop the commit ...

Reply:
No we definitely need to reduce the size... So good for me